### ISSUE TITLE
`checkbox_tree` do not check existing value / do not save values

### ISSUE THREAD

**Issue by francoismassart** [wpmetabox/meta-box#1289]
## Issue Overview
I did the plugin update from `meta-box.4.15.5` to `meta-box.4.15.6`, since I did it, when I edit a custom post type which uses a custom taxonomy with a `checkbox_tree` the checked values disappeared!

If I do some changes in the tree of checkboxes, they are not saved either!

It was all fine in the previous minor version (4.15.5)...

## Code that register meta boxes

It used to work fine but since the update `4.15.6` the previously saved values are neither shown or saved.

```
/**
 * Detailed Activities MetaBox
 */

function dwkw_detailed_activities_meta_boxes( $meta_boxes ) {

    $meta_boxes[] = array(
        'id'         => 'meta-box-detailed-activities',
        'title'      => __( 'Activity', 'dreamwall-keywall-plugin' ),
        'post_types' => ['dwkw_videowall_brick'],
        'fields'     => array(
            // List all the activities
            array(
                'id'   => 'detailed_activities',
                'name' => __( 'Activities taxonomy', 'dreamwall-keywall-plugin' ),
                'desc' => __( 'Used on the VideoWall for concise filtering.', 'dreamwall-keywall-plugin' ),
                'type' => 'taxonomy',
                'taxonomy' => 'activity_taxonomy',
                'field_type' => 'checkbox_tree',
            ),
        ),
    );

    return $meta_boxes;
}

add_filter( 'rwmb_meta_boxes', 'dwkw_detailed_activities_meta_boxes' );
```
## Expected Behavior
When I edit my content I should see the previously saved values... 
When I save it should save...

## Current Behavior
When I load the modify page, all checkboxes are unchecked.
After a save, it does not remember the checked options.

## Possible Solution
Looks like there is a problem for retrieving and saving.

### `meta-box.4.15.5` works fine

![4 15 5](https://user-images.githubusercontent.com/704026/48189897-847d8c80-e341-11e8-8672-28125f750cc6.gif)

### `meta-box.4.15.6` does not show or save my taxonomies

![4 15 6](https://user-images.githubusercontent.com/704026/48189898-847d8c80-e341-11e8-91fb-61ddfd5bcfd0.gif)

**Comment by francoismassart** [comment]
Using `WordPress 4.9.8`

**Comment by rilwis** [comment]
HI @francoismassart , this is fixed in this commit efd24ca0e8f632ea854e9bb3c35c3044373131f2. Can you please check?

**Comment by francoismassart** [comment]
@rilwis Thank you for your help, I did test it via release `4.15.7` and it works fine 👍
